tests/test_movies.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+
+from moviecenter import movies
+from moviecenter.movies import Movie
+
+
+@pytest.fixture
+def test_movie_1() -> Movie:
+    return Movie(name='LoTR', director='Steve', budget=100000000)
```

Comment:
nit: I like to use the `_` number separator syntax for numbers greater than 1000: `100_000_000`.
In that case you could also use the scientific notation (`1E8` or `100E6`), but that's less readable IMHO and would yield a `float` and not an `int`